registration/migrations/0104_auto_20241114_1419.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 3.2.25 on 2024-11-14 19:19
+
+from django.contrib.postgres.operations import TrigramExtension
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
OK for these but going forward let's give migrations descriptive names with e.g. `./manage.py makemigrations -name trigram_extension`